From 961269271134e711bcfffebb0f179ffddcbf3d5a Mon Sep 17 00:00:00 2001
From: Francois Perrad <francois.perrad@gadz.org>
Date: Sun, 29 Jul 2018 11:17:34 +0200
Subject: [PATCH] allow libluajit detection

This detection was done only if luarocks is runned by luajit.
But on Buildroot, luarocks is always runned by lua.

See https://github.com/luarocks/luarocks/pull/883

Signed-off-by: Francois Perrad <francois.perrad@gadz.org>
---
 src/luarocks/deps.lua | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/src/luarocks/deps.lua b/src/luarocks/deps.lua
index 8403f12..c1c0220 100644
--- a/src/luarocks/deps.lua
+++ b/src/luarocks/deps.lua
@@ -513,10 +513,8 @@ function deps.check_lua(vars)
       "lua-" .. cfg.lua_version,
       "lua-" .. shortv,
       "lua",
+      "luajit-" .. cfg.lua_version,
    }
-   if cfg.luajit_version then
-      table.insert(libnames, 1, "luajit-" .. cfg.lua_version)
-   end
    for _, libname in ipairs(libnames) do
       local ok = check_external_dependency("LUA", { library = libname }, vars, "build")
       if ok then
-- 
2.17.1

